src/App.css

```diff
@@ -1,3 +1,29 @@
 #root {
   height: 100vh;
 }
+.modal {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.25);
+}
+
+.modal_content {
+  position: absolute;
+  background-color: white;
+  position: abolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
As mentioned during the demo, I love the combination of `top`/`left` and `transform`. That's a clever way to make sure the modal will always be centered. How does it look on a mobile screen, though? Modals can be tricky on smaller devices.